### PR TITLE
Changing readme to prepare to release to central

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
1) Changed readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,20 +14,22 @@
...
-1. Install KTlint (until this [PR](https://github.com/pinterest/ktlint/pull/806) is merged you will need to use
- [KTlint fork](https://central.artipie.com/akuleshov7/files/ktlint)):
-   ```bash
-   $ curl -sSLO https://central.artipie.com/akuleshov7/files/ktlint && chmod a+x ktlint
-   ```
+1. Install KTlint manually: [here](https://github.com/pinterest/ktlint/releases)
+
+   **OR** use curl:
+    ```bash
+    curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.37.1/ktlint && chmod a+x ktlint
```
Comment (petertrr) [inline README.md:30]: KtLint 0.38+ uses kotlin 1.4 and is not yet compatible with diktat

File: README.md [hunk 2]
```diff
@@ -100,7 +84,7 @@ Then, add this plugin:
               <dependency>
                   <groupId>com.pinterest</groupId>
                   <artifactId>ktlint</artifactId>
-                  <version>0.37.1-fork</version> <!-- use this fork to be compatible with diktat -->
+                  <version>0.37.1</version> <!-- use this fork to be compatible with diktat -->
```
Comment (petertrr) [inline README.md:87]:
```suggestion
                  <version>0.37.1</version>
```

File: README.md (hunk 1 above)
Comment (akuleshov7) [reply README.md:30]: Yes, I know, Amended yesterday before your comment :)

File: README.md (hunk 2 above)
Comment (akuleshov7) [reply README.md:87]: Same :)

**Review by petertrr — APPROVED** [review]
Right, haven't seen your amend :) LGTM

**Pull request merged**